selfdrive/car/toyota/carcontroller.py

```diff
@@ -121,14 +125,19 @@ def update(self, CC, CS, now_nanos):
     if not CC.enabled and CS.pcm_acc_status:
       pcm_cancel_cmd = 1
 
-    # on entering standstill, send standstill request
-    if CS.out.standstill and not self.last_standstill and (self.CP.carFingerprint not in NO_STOP_TIMER_CAR or self.CP.enableGasInterceptor):
-      self.standstill_req = True
-    if CS.pcm_acc_status != 8:
-      # pcm entered standstill or it's disabled
-      self.standstill_req = False
```

Comment:
Why remove this?

Reply:
It seems unnecessary, we can request standstill even if cruise is off and it wouldn't cause any issues.